### PR TITLE
chore: move pending-release to release-notes

### PR DESCRIPTION
**Pull request by rootulp**
Also adds a line for node operators to modify their mempool ttl-num-blocks config

### CODE REVIEW

**Review by ninabarbakadze — APPROVED** [review]

**Review by evan-forbes — APPROVED** [review]

File: docs/release-notes/release-notes.md [hunk 1]
```diff
@@ -4,19 +4,34 @@ This guide provides notes for major version releases. These notes may be helpful
...
 If you are a library consumer, a number of the Go APIs have changed since celestia-app v1.x.x. Some of the notable changes are:
 
 - Code pertaining to the original data square was extracted to [celestiaorg/go-square](https://github.com/celestiaorg/go-square).
   - celestia-app v1.x had a shares package. celestia-app v2.x uses [go-square/shares](https://github.com/celestiaorg/go-square/tree/c8242f96a844956f8d1c60e5511104deed8bc361/shares)
   - celestia-app v1.x had a blob.types package with `CreateCommitment` function. celestia-app v2.x uses `CreateCommitment` function from the [go-square/inclusion](https://github.com/celestiaorg/go-square/tree/c8242f96a844956f8d1c60e5511104deed8bc361/inclusion).
 - celestia-app v1.x had a lot of functionality included in the signer. celestia-app v2.x splits a txClient from the signer. See [#3433](https://github.com/celestiaorg/celestia-app/pull/3433).
+
+## [v3.0.0-rc0](https://github.com/celestiaorg/celestia-app/releases/tag/v3.0.0-rc0)
+
+### Node Operators (v3.0.0-rc0)
```
Comment (evan-forbes) [inline docs/release-notes/release-notes.md:26]: I'd be fine just saying v3.0.0, or `all v3.*+` tags or something so we don't have to update this constantly. ie we're going to cut the arabica tag soon

Comment (rootulp) [reply docs/release-notes/release-notes.md:26]: Agreed, will change

File: docs/release-notes/release-notes.md [hunk 2]
```diff
@@ -4,19 +4,34 @@ This guide provides notes for major version releases. These notes may be helpful
...
+### Library Consumers (v2.0.0)
 
 If you are a library consumer, a number of the Go APIs have changed since celestia-app v1.x.x. Some of the notable changes are:
 
 - Code pertaining to the original data square was extracted to [celestiaorg/go-square](https://github.com/celestiaorg/go-square).
   - celestia-app v1.x had a shares package. celestia-app v2.x uses [go-square/shares](https://github.com/celestiaorg/go-square/tree/c8242f96a844956f8d1c60e5511104deed8bc361/shares)
   - celestia-app v1.x had a blob.types package with `CreateCommitment` function. celestia-app v2.x uses `CreateCommitment` function from the [go-square/inclusion](https://github.com/celestiaorg/go-square/tree/c8242f96a844956f8d1c60e5511104deed8bc361/inclusion).
 - celestia-app v1.x had a lot of functionality included in the signer. celestia-app v2.x splits a txClient from the signer. See [#3433](https://github.com/celestiaorg/celestia-app/pull/3433).
+
+## [v3.0.0-rc0](https://github.com/celestiaorg/celestia-app/releases/tag/v3.0.0-rc0)
```
Comment (rootulp) [inline docs/release-notes/release-notes.md:24]:
```suggestion
## v3.0.0
```

File: docs/release-notes/release-notes.md (hunk 1 above)
Comment (rootulp) [inline docs/release-notes/release-notes.md:26]:
```suggestion
### Node Operators (v3.0.0)
```

File: docs/release-notes/release-notes.md [hunk 3]
```diff
@@ -4,19 +4,34 @@ This guide provides notes for major version releases. These notes may be helpful
...
+### Node Operators (v3.0.0-rc0)
+
+- Consensus node operators must enable the BBR (Bottleneck Bandwidth and Round-trip propagation time) congestion control algorithm. See [#3774](https://github.com/celestiaorg/celestia-app/pull/3774).
+- Consensus node operators should manually configure their node's mempool `ttl-num-blocks = 12`. An example command to do this:
+
+  ```bash
+  sed -i 's/ttl-num-blocks = 5/ttl-num-blocks = 12/' ~/.celestia-app/config/config.toml
+  ```
+
+### Library Consumers (v3.0.0-rc0)
```
Comment (rootulp) [inline docs/release-notes/release-notes.md:35]:
```suggestion
### Library Consumers (v3.0.0)
```

**Review by evan-forbes — APPROVED** [review]

**Review by cmwaters — CHANGES_REQUESTED** [review]
I would prefer if the latest release was always put at the top. 

And we should not remove the `pending-release` document but rather update it to v4

**Review by evan-forbes — APPROVED** [review]

File: docs/release-notes/release-notes.md [hunk 4]
```diff
@@ -2,17 +2,32 @@
...
+## v3.0.0
+
+### Node Operators (v3.0.0)
+
+- Consensus node operators must enable the BBR (Bottleneck Bandwidth and Round-trip propagation time) congestion control algorithm. See [#3774](https://github.com/celestiaorg/celestia-app/pull/3774).
+- Consensus node operators should manually configure their node's mempool `ttl-num-blocks = 12` in config.toml. An example command to do this:
+
+  ```bash
+  sed -i 's/ttl-num-blocks = 5/ttl-num-blocks = 12/' ~/.celestia-app/config/config.toml
+  ```
```
Comment (cmwaters) [inline docs/release-notes/release-notes.md:14]: Does this need to be indented?

**Review by cmwaters — APPROVED** [review]

File: docs/release-notes/release-notes.md (hunk 4 above)
Comment (rootulp) [reply docs/release-notes/release-notes.md:14]: I think desirable for the code block to be indented so it appears under the bullet point that it's related to

Screenshot 2024-10-22 at 11 58 22 AM

**Pull request merged**